pkg/storage/backend-storage/backend-storage.go

```diff
@@ -118,3 +121,42 @@ func CreateIfNeeded(vmi *corev1.VirtualMachineInstance, clusterConfig *virtconfi
 
 	return err
 }
+
+// IsPVCReady returns true if either:
+// - No PVC is needed for the VMI since it doesn't use backend storage
+// - The backend storage PVC is bound
+// - The backend storage PVC is pending uses a WaitForFirstCustomer storage class
+func IsPVCReady(vmi *corev1.VirtualMachineInstance, client kubecli.KubevirtClient, scInformer cache.SharedIndexInformer) (bool, error) {
+	if !IsBackendStorageNeededForVMI(&vmi.Spec) {
+		return true, nil
+	}
+
+	pvc, err := client.CoreV1().PersistentVolumeClaims(vmi.Namespace).Get(context.Background(), PVCForVMI(vmi), metav1.GetOptions{})
```

Comment:
Should be a pvc informer in `VirtControllerApp` as well sorry missed this last time

Reply:
That was discussed above: https://github.com/kubevirt/kubevirt/pull/11373#discussion_r1505075891
Since the function gets called right after creating the PVC, I'm worried the informer might not have it yet. That would be an expensive scenario. WDYT?

Reply:
Ah okay that makes sense and since this function is only called when the VMI pod does not exist I think the extra overhead of going to the k8s apiserver in this case is no big deal